docs(contributions): add contribution guidelines and a partner list

rasos opened the pull request:
### What does this PR do?

Adds a guideline for financial contributions and a partner list and how to get on the list. 

### Motivation

At the BBB Dev Summit in Vienna in November 2025 we had discussed options how to motivate organisations and companies that use BigBlueButton to contribute financially to the software ecosystem. A proposal for a clear guideline at https://hedgedoc.uni-osnabrueck.de/fairpay  has been circulated in the participants' chat group. This is a slightly improved version.

rasos committed:
Add FAIRPAY.md with contribution guidelines

This document outlines the expectations for organizations using the software to contribute back to the project, including financial recommendations.

rasos committed:
Add partner and contributor sections in PARTNERS.md

Add partner list

rasos committed:
Revise FAIRPAY.md for clarity and budget recommendations

Updated the FAIRPAY.md document to include a new section on fair budget allocation and modified the language for clarity.

rasos commented:
A CLA has been signed in 2023 by fairkom, but I will send in a personal for user @rasos .

antobinary reviewed (approved):
Thank you @rasos !

I read through, looks good to me!

antobinary commented on FAIRPAY.md:
```diff
@@ -0,0 +1,20 @@
+## Pay the Developers
+
+This software has been developed through the considerable effort of its contributors. While free software guarantees your liberty to run, study, modify, and share it (refer to LICENSE.TXT), its long-term health depends on a reciprocal ecosystem.
```
```suggestion
This software has been developed through the considerable effort of its contributors. While free software guarantees your liberty to run, study, modify, and share it (refer to the LICENSE file), its long-term health depends on a reciprocal ecosystem.
```

A tiny tweak - BBB's license file in this repository does not have a file extension and it seems to be standard GitHub pattern from what I see.

g-v-egidy commented:
Just a comment from a user in an organization that might want to contribute financially:

The sections "Fair Budget" and "Recommended Financial Contributions" clearly state what is expected.

But what is a bit vague is how to do that. 

Is hiring one of the listed partner companies the only way? How to contribute financially when you don't need any consulting, hosting and such?

You want financial support, so make it easy to give it without requiring guesswork, complicated processes or contacting sales.

rasos commented:
@g-v-egidy good point - I will add such links for a one-time and for an annual contribution to the partner I am affiliated with. Each partner may add a link how to contribute or donor directly as an extra PR.

rasos committed:
Correct license reference in FAIRPAY.md

Updated the reference from LICENSE.TXT to LICENSE for consistency.

rasos committed:
Update financial contribution fees to EUR

As drafted at the BBB dev summit, and as we have many BBB deployments in the EU, we show EUR

rasos committed:
Update additional fee information

Added links to shop for one-time and annual fees for partner fairkom

rasos committed:
Clarify contribution expectations and fix typos

Updated language for clarity and corrected formatting.

ffdixon commented:
Hi @rasos, thanks for the input.  Good to see you at the summit!.

We're working on bigbluebutton.com, which will coordinate the commercial side of BigBlueButton.  We'll hold this pull request open until we launch that site (soon!) and then adjust this pull request to ensure there is no overlap.
